src/api/app/models/package.rb

```diff
@@ -1374,4 +1361,19 @@ def file_exists?(filename)
   def has_icon?
     file_exists?("_icon")
   end
+
+  private
+
+  # is called before_update
+  def update_activity
+    # the value we add to the activity, when the object gets updated
+    addon = 10 * (Time.now.to_f - updated_at_was.to_f) / 86400
+    addon = 10 if addon > 10
+    new_activity = activity + addon
+    new_activity > 100 ? 100 : new_activity
+
+    self.activity_index = new_activity
+    # just for beauty
+    self.update_counter += 1
```

Comment:
cdfaf4d62fdc4c9554ec80dcc63905e658794964 is 4 years old - and beauty lies in the eye of the beholder :)

But the whole point of the activity index is that this is counting real activity - not clicking around in repository flags. But again: I only fixed in 2012 what we added much earlier - and IMO this never was used. I experimented a bit back then with the data and realized quickly that bots are much more active than humans, so the whole point of this index for larger instances is mood. Unless you want to display it in the project page